frontend/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "vue-update",
+  "description": "Vue Auth & more",
+  "version": "1.0.0",
+  "author": "Maximilian Schwarzmüller <business@academind.com>",
+  "private": true,
+  "scripts": {
+    "dev": "cross-env NODE_ENV=development webpack-dev-server --open --hot",
```

Comment:
Z tego co wiem, to praktyka jest taka, że ten skrypt powinien się nazywać start i wtedy apkę do developmentu odpala się przez "npm start"